common/usermode.c

```diff
@@ -89,6 +92,9 @@ static void init_syscall(void) {
 
 void init_usermode(percpu_t *percpu) {
     vmap_user_4k(&cr3, virt_to_mfn(&cr3), L1_PROT);
+    vmap_user_4k(&user_cr3, virt_to_mfn(&user_cr3), L1_PROT);
+    vmap_user_4k(&exception_handlers, virt_to_mfn(&exception_handlers), L1_PROT);
+    vmap_user_4k(&interrupt_handlers, virt_to_mfn(&interrupt_handlers), L1_PROT);
```

Comment:
Might make sense to add asserts that these don't get larger than a page.

Reply:
Yes, I will do it.